main_check.go

```diff
@@ -26,8 +26,8 @@ func GitSeekretCheck(c *cli.Context) error {
 		options["commit-count"] = c.Int("commit")
 	}
 
-	if c.IsSet("staged") {
```

Comment:
When would you actually use this flag? Does this make sense for the user at all?

How about, we just leave all the flags alone as they are, We just maybe update the `--help` to say, when you provide no arguments, `-s`, or staged files, shows by default. You still leave `-s` in place in case they want to actually use it, even if they don't technically have to.